README.md

```diff
@@ -228,7 +228,7 @@ The admin api only allows the creating of users using the email/password sign-in
 ```
 
 ### Working with Firestore (Alpha)
-The the API allows for CRUB operation on firestore.
+The the API allows for CRUD operation on firestore.
```

Comment:
@bherrmann7 could you also fix the duplicate the at the beginning of the sentence. Then I'll do a merge.